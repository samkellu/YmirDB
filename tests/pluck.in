pluck a 1
set a
pluck a
set a 123 22
pluck a 5
get a
pluck a 1
set b a 22 45 56
pluck b 1
get b
get a
bye
