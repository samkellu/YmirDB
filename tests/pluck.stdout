==822== Memcheck, a memory error detector
==822== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==822== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==822== Command: ./ymirdb
==822== 
==822== error calling PR_SET_PTRACER, vgdb might block
> no such key

> ok

> > ok

> index out of range

> [123 22]

> ok

> a

> [22 45 56]

> [123 22]

> bye
==822== 
==822== HEAP SUMMARY:
==822==     in use at exit: 0 bytes in 0 blocks
==822==   total heap usage: 52 allocs, 52 frees, 2,304 bytes allocated
==822== 
==822== All heap blocks were freed -- no leaks are possible
==822== 
==822== For lists of detected and suppressed errors, rerun with: -s
==822== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
